pybaseball/statcast_fielding.py

```diff
@@ -55,3 +55,18 @@ def statcast_catcher_framing(year: int, min_called_p: Union[int, str] = "q") ->
 	# CSV includes league average player, which we drop from the result
 	return data.loc[data.last_name.notna()].reset_index(drop=True)	
 
+@cache.df_cache()
+def statcast_arm_strength(year: int, pos: Union[int, str], min_throws: int = 100):
```

Comment:
I think you've got `pos` and `min_throws` in a different order than in the readme or the tests. 
`

Reply:
That seems to have been it, but still failing the tests. I searched the logs and didn't find any mention of failing this particular test. Am I missing other changes or something?